biogeochem/EDPhysiologyMod.F90

```diff
@@ -453,8 +503,8 @@ subroutine trim_canopy( currentSite )
              lai_current       = min(leaf_inc, currentCohort%treelai - lai_layers_above)
              cumulative_lai    = lai_canopy_above + lai_layers_above + 0.5*lai_current
              
-             if (currentCohort%year_net_uptake(z) /= 999._r8)then !there was activity this year in this leaf layer.
-             
+             !there was activity this year in this leaf layer.  This should only occur for bottom most leaf layer
+             if (currentCohort%year_net_uptake(z) /= 999._r8)then 
```

Comment:
These clauses where we compare real numbers for equivalence seem dangerous to me.  But as far as I know, the compilers seem to get it right as long as year_net_uptake is set with a constant and not the result of math. I'm always in favor of this style:   
> if( abs(this - that) > nearzero ) then

Reply:
Per our discussion, I created issue #644 to resolve this in a future, more broad canopy trimming refactor.